Add alias for translations of script_extender_plugin_checker.

Holt59 opened the pull request:
The slug of the translations is `ScriptExtenderPluginChecker` on Transifex to match the filename of the plugin, this just removes the `mob` warning about no matching task.

isanae commented:
Did this change? I'm pretty sure I added `diagnose-script_extender_plugin_checker` because that was the slug on tx.

Holt59 commented:
> 
> 
> Did this change? I'm pretty sure I added `diagnose-script_extender_plugin_checker` because that was the slug on tx.

Yes I changed it because the filename is `ScriptExtenderPluginChecker` so the translations were not loaded by MO2.

isanae commented:
Can you remove the other one?

Holt59 commented:
> 
> 
> Can you remove the other one?

Done.
